net-im/swift/swift-4.0.2-r2.ebuild

```diff
@@ -0,0 +1,208 @@
+# Copyright 1999-2019 Gentoo Authors
```

Comment:
why did you revbump? can't you just attach it to the older ebuild?

Reply:
Yes. I've forgotten, that due slot operator, swift will rebuild anyway. So yes :) Thanks.